module-2/backend/week_one.md

```diff
@@ -1,32 +1,146 @@
-## Week One - Module 2 Recap
-
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
-
-Note: When you're done, submit a PR. 
-
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
-  ```ruby
-  get '/horses' do
-    erb :index
+Week One - Module 2 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
+
+Note: When you're done, submit a PR.
+
+List the five common HTTP verbs and what the purpose is of each verb.
+1. Get - retrieve data
+* POST - sending data to the server
+* DELETE - remove data
+* PUT - will replace/update
+* cant think of another :(
+
+
+What is Sinatra?
+* Sinatra enables us to have a client server relationship. It is usually used for creating simple web applications.
+
+What is MVC?
+* Model/View/Controller
+
+Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+* The purpose of following conventions is so that someone else could pick up your application and understand the code/syntax.
+
+What types of variables are accessible in our view templates without explicitly passing them?
+* Instance variables
+
+Given the following block of code, how would I pass an instance variable count with a value of 1 to my index.erb template?
+
+```
+get '/horses' do
+  @count = 1
+  name = "Mr. Ed"
+  erb :index, :local => {name: name}
+end
+```
+
+In the same code block, how would I pass a local variable name with a value of Mr. Ed?
+
+
+What's the purpose of ERB?
+* Erb lets us display our data to the client. It also has the ability to read ruby code when you use special syntax
+
+Why do I need a development AND test database?
+* The test database is for test and it is destroyed right after where as a development database is for non-testing use so it is not destroyed unless given a command

```

Comment:
Good answer. One note: the test database is only destroyed after if we specifically make it so thats the case (e.g. by using a gem like `database_cleaner`)